packages/accounts-controller/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.1]
+### Changed
+- Update TypeScript to v4.8.x ([#1718](https://github.com/MetaMask/core/pull/1718))
```

Comment:
I was unsure whether to include these changes, as they don't _really_ impact users of the package in most cases. Sometimes TypeScript updates result in meaningful differences in generated JS and types, but I'm not sure if this one does.

But it would greatly simplify future releases to ship these changes in some form, and this was the most relevant change to document. Plus better safe than sorry I guess, in case this does end up mattering.